### ISSUE TITLE
Master branch is broken?

### ISSUE THREAD

**Issue by dorner** [patrickleet/apply-template#18]
Tried running this locally and nothing works. :( Tried to do this as a meta plugin and couldn't find any documentation, so I tried running it as a command line. Trying to create a new plugin called meta-config:

`./apply-template ../../../../meta-template-meta-plugin  ../../meta-config`

1) This couldn't find variables, since the meta-template-meta-plugin template has not changed but the parser has
2) Putting the parser back the way it was prompted for variable values - as soon as one variable is read, the command just ends/dies
3) Passing the variables in via the command-line, the files get copied but none of the variables are replaced.

Any ideas?

**Comment by patrickleet** [comment]
hey @dorner - so I gave up on making meta-template a meta plugin, hence the separation there, as it wasn't adding much benefit to this project - meta is strong when looping through projects, which this didn't do.

I changed the parser delimiters because I ran across a project which had minified code that was matching the delimiters when it was not supposed to be.

The `meta-plugin` template I made was last updated over a year ago, so I think it's probably best to update the template and then use `apply-template` as it is. It just needs the new delimiters.

Recently another change was introduced which fixed one problem but added another, and I never got around to attempting to fix it.

https://github.com/patrickleet/apply-template/commit/b106216a6dd801a019619e032b88327d12e3848d

Which was that ^

TBH I didn't think anyone was really using this anymore. I'll pull it up later and try to get it going again.

**Comment by patrickleet** [comment]
this worked a lot better before I accepted PRs - seeing same issue where only one input is accepted now as well with updated delimeters

**Comment by patrickleet** [comment]
I think a rewrite with aysnc/await instead  of generators would make it less confusing

**Comment by dorner** [comment]
Agreed - generators confuse the hell out of me :p 
I'm fine without using the templates - there are just a lot of references to it in the documentation. If you excise it and remove the templates from meta, I'd be fine with that 👍 

I'm going to add a new plugin and some PRs to this - I think meta has the ability to be a really powerful tool for the microservice programming paradigm we use (basically treat the parent as the "application" and the sub-repos as "services"). Developing locally is a giant pain because of how meta detects plugins though :(

**Comment by patrickleet** [comment]
Yes I use meta for microservices as well. Also helm. Both for representing
bounded contexts. Helm for production, meta for local.

**Comment by patrickleet** [comment]
Developing what locally is difficult? Plugins? You’ll need npm link.

**Comment by dorner** [comment]
I've tried with that. meta is looking in the wrong spot - it's looking in the node_modules directory of my local meta project, which is not where my new plugin lives. I had to change it manually in that directory. And every time I change something it seems I have to npm link again, which takes about a minute because of all the different plugins. I'm relatively new to node.js though so I'm likely doing something wrong. :) I did figure it out and I'll hopefully have some PRs for you tomorrow.

Not sure how to do a PR when I have a brand new plugin which I think should be part of the base meta package :)
